__tests__/utility.service.test.js

```diff
@@ -19,18 +31,21 @@ describe("UtilityService", () => {
   });
 
   describe("updateSearchResults", () => {
-    it("should call provided functions", () => {
+    it("should call provided functions and ", () => {
       let mockSetSearchResults = jest.fn();
       let mockSetDisplayValue = jest.fn();
 
       utilityService.updateSearchResults(
         "",
         mockSetSearchResults,
-        mockSetDisplayValue
+        mockSetDisplayValue,
+        mockCurrentLocation
       );
 
       expect(mockSetSearchResults).toHaveBeenCalledTimes(1);
       expect(mockSetDisplayValue).toHaveBeenCalledTimes(1);
+      expect(mockCurrentLocation).not.toBe(null);
+      expect(mockCurrentMarkerLocation).not.toBe(null);
```

Comment:
These tests are a bit useless since you're the one controlling these inputs.

Reply:
Removed useless tests.